src/System.Threading.Timer/tests/TimerFiringTests.cs

```diff
@@ -75,46 +77,13 @@ public void Timer_FiresOnlyOnce_OnDueTime_With_InfinitePeriod()
             }
         }), null, TimeSpan.FromMilliseconds(100), TimeSpan.FromMilliseconds(Timeout.Infinite) /* not relevant */))
         {
-            Assert.False(are.WaitOne(TimeSpan.FromSeconds(1)));
-        }
-    }
-
-    [Fact]
-    public void MultipleTimers_ShouldFire_InCorrectOrder()
-    {
-        object t1State = new object();
-        object t2State = new object();
-        Timer t1 = null;
-        Timer t2 = null;
-        int count = 0;
-        AutoResetEvent are = new AutoResetEvent(false);
-        TimerCallback tc = new TimerCallback((object o) =>
-        {
-            if (count == 0)
-            {
-                Assert.Same(t2State, o);
-                count++;
-            }
-            else
-            {
-                Assert.Same(t1State, o);
-                are.Set();
-            }
-        });
-
-        using (t1 = new Timer(tc, t1State, TimeSpan.FromSeconds(1), TimeSpan.FromMilliseconds(-1)))
-        using (t2 = new Timer(tc, t2State, TimeSpan.FromMilliseconds(0), TimeSpan.FromMilliseconds(-1)))
-        {
-            // Wait for both events to fire
-            Assert.True(are.WaitOne(TimeSpan.FromMilliseconds(1500 /*1.5 seconds*/)));
+            Assert.False(are.WaitOne(TimeSpan.FromMilliseconds(MaxPositiveTimeoutInMs)));
```

Comment:
I was suggesting MaxPositiveTimeoutInMs be used in places of the form ```Assert.True(are.WaitOne(...))``` because in those cases we're expecting an event and the only reason we have a timeout at all is to avoid hanging the tests in the case of failure.  But this is ```Assert.False(are.WaitOne(...))``` because we're expecting that we *won't* get an event, which means that in a successful run we'll end up waiting for the entire timeout. Such cases should not use a long value, but should use the shortest value that's reasonable.